be2-scala/src/main/scala/ch/epfl/pop/storage/DbActor.scala

```diff
@@ -74,8 +74,8 @@ final case class DbActor(
   }
 
   @throws[DbActorNAckException]
-  private def readElectionData(electionId: Hash): ElectionData = {
-    Try(storage.read(s"${ROOT_CHANNEL_PREFIX}private/${electionId.toString}")) match {
+  private def readElectionData(electionId: Hash, laoId: Hash): ElectionData = {
```

Comment:
For more clarity, maybe you should also switch the paramters here in the signature. There will be less confusion when you use the `ReadElectionData`  function